### PR TITLE
yocto/init_ws_ids: Support TRUSTME_PLAIN_DATAPART

### PR DESCRIPTION
**Pull request by k0ch4lo**
This PR introduces support for the variable TRUSTME_PLAIN_DATAPART introduced in PR gyroidos/meta-trustx#232

### CODE REVIEW

File: yocto/init_ws_ids.sh [hunk 1]
```diff
@@ -81,6 +81,15 @@ if ! grep -q '##TRUSTME_HARDWARE##' ${BUILD_DIR}/conf/local.conf;then
 	       echo "Not enabling sanitizers for cmld"
 	       sed -i 's/##TRUSTME_SANITIZERS##/n/' ${BUILD_DIR}/conf/local.conf
 	fi
+
+	if [ "${TRUSTME_PLAIN_DATAPART}" = "1" ]; then
+	       echo "Enabling sanitizers for cmld"
```
Comment (quitschbo) [inline yocto/init_ws_ids.sh]: s/sanitizers/plain data partition/

File: yocto/init_ws_ids.sh [hunk 2]
```diff
@@ -81,6 +81,15 @@ if ! grep -q '##TRUSTME_HARDWARE##' ${BUILD_DIR}/conf/local.conf;then
...
+	       sed -i 's/##TRUSTME_PLAIN_DATAPART##/y/' ${BUILD_DIR}/conf/local.conf
+	else
+	       echo "Not enabling sanitizers for cmld"
```
Comment (quitschbo) [inline yocto/init_ws_ids.sh]: s/sanitizers/plain data partition/

**Review by quitschbo — DISMISSED** [review]

File: yocto/init_ws_ids.sh (hunk 1 above)
Comment (k0ch4lo) [reply yocto/init_ws_ids.sh]: done

File: yocto/init_ws_ids.sh (hunk 2 above)
Comment (k0ch4lo) [reply yocto/init_ws_ids.sh]: done

**Review by jwsbck — APPROVED** [review]

**Pull request merged**